mava/systems/jax/ippo/system.py

```diff
@@ -94,3 +94,71 @@ def design(self) -> Tuple[DesignSpec, Any]:
             component_dependency_guardrails=ComponentDependencyGuardrails,
         )
         return system, default_params
+
```

Comment:
Change the name of the file to systems or separate the two systems each in a py file since we have a full folder called systems

Reply:
The extra system has been removed